tests/fixtures/utils.py

```diff
@@ -1,12 +1,12 @@
-from ml_warehouse.schema import (
+from sqlalchemy import delete, text
```

Comment:
`text` seems redundant.

Reply:
Thanks, removed in https://github.com/wtsi-npg/npg_langqc/pull/138/commits/9d48fd9808fcd4e23dddc5c6d4c6a6e5385b2979